### PR TITLE
ddl: add limitation for index comment

### PR DESCRIPTION
**Pull request by jackysp**
Fix issue #4618. @zimulala PTAL

### CODE REVIEW

File: parser/parser.y [hunk 1]
```diff
@@ -2234,6 +2234,9 @@ IndexOption:
 	}
 |	"COMMENT" stringLit
 	{
+		if len($2) > 1024 {
```
Comment (shenli) [inline parser/parser.y]: Can you define a const in mysql/const.go ?

**Comment by shenli** [comment]
In MySQL this is not a syntax error.

**Comment by jackysp** [comment]
Use MySQL style to deal with this error. PTAL @shenli

**Comment by coocood** [comment]
LGTM

**Comment by shenli** [comment]
LGTM

**Comment by jackysp** [comment]
CI failed, please wait.

**Pull request merged**